system-reinstall-bootc/src/podman.rs

```diff
@@ -41,6 +41,9 @@ pub(crate) fn reinstall_command(image: &str, ssh_key_file: &str) -> Command {
         // this binary. Since this is no longer an "arcane" bootc command, we can safely avoid this
         // timed warning prompt. TODO: Discuss in https://github.com/containers/bootc/discussions/1060
         "--acknowledge-destructive",
+        // The image is always pulled into containers-storage first, so let's use it here to avoid
+        // requiring registry keys to be baked into the image.
+        "--target-transport containers-storage",
```

Comment:
Because this fundamentally changes what it does. Previously we'd look for updates from the registry, now we won't - to do an upgrade the admin would need to `podman pull` subsequent images which would be a huge change.

Reply:
ah, right, I forgot that flag will affect upgrades. I switched to use `skip-fetch-check` for now and can follow up to make it the default.

Reply:
Of course, really this is all extremely confusing and https://github.com/containers/bootc/issues/20 would help a whole lot in all of this